### PR TITLE
Bring back consul agent

### LINKED ISSUE #2: Remove consul agent
Any reason you removed the consul agent coprocess? It seems like that is the way to go and what all the other example autopilotpattern projects are doing...

commit: e03775d618e3a49530662c34b48b237aaa1136b2

### PR DESCRIPTION
**Pull request by sberryman**
Fixes #2

### CODE REVIEW

**Comment by sberryman** [comment]
Still have some questions... 

* What happens if you scale up `influxdb=2` etc?
* Thoughts about backup and restore? I'm assuming this should utilize Manta?

File: etc/containerpilot.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "consul": "{{ .CONSUL }}:8500",
+  "consul": "localhost:8500",
```
Comment (geek) [inline etc/containerpilot.json]: We will want to check the env variable CONSUL_AGENT to determine where to connect to..

```
"consul": "{{ if .CONSUL_AGENT }}localhost{{ else }}{{ if .CONSUL }}{{ .CONSUL }}{{ else }}consul{{ end }}{{ end }}:8500",
```

File: etc/containerpilot.json [hunk 2]
```diff
@@ -11,5 +11,17 @@
       "poll": 5,
       "ttl": 25
     }
+  ],
+  "coprocesses": [
```
Comment (geek) [inline etc/containerpilot.json]: We will want to check the env variable CONSUL_AGENT here too

```
"coprocesses": [{{ if .CONSUL_AGENT }}
    {
      "command": ["consul", "agent",
                  "-data-dir=/var/lib/consul",
                  "-config-dir=/etc/consul",
                  "-rejoin",
                  "-retry-join", "{{ if .CONSUL }}{{ .CONSUL }}{{ else }}consul{{ end }}",
                  "-retry-max", "10",
                  "-retry-interval", "10s"],
      "restarts": "unlimited"
    }{{ end }}
```

**Comment by sberryman** [comment]
Got it, i'll add in the optional agent stuff

**Comment by geek** [comment]
@sberryman you rock!

**Comment by geek** [comment]
@sberryman this is now published as _autopilotpattern/influxdb:1.2.2-r1.0_ and _autopilotpattern/influxdb:latest_

**Pull request merged**